docs/pages/setup/reference/metrics.mdx

```diff
@@ -34,6 +32,24 @@ The response body is a JSON object of the form:
 { "status": "a status message here"}
 ```
 
+### `/readyz` and heartbeats
+
+Teleport heartbeats run every 5 seconds. This means that depending on the timing
+of heartbeats, it can take 10-20 seconds after connectivity is restored for
+`/readyz` to start reporting healthy again.
+
+If a Teleport component fails to execute its heartbeat procedure, it will enter
```

Comment:
Shouldn't this paragraph come first?

I would think we want to talk about how Teleport becomes unhealthy before we talk about how it recovers.